.mergify.yml

```diff
@@ -1,10 +1,16 @@
+queue_rules:
+  - name: default
+    conditions:
+      - check-success=build
+      - check-success=DeepScan
+      
 pull_request_rules:
   - name: Automatic merge on approval
     conditions:
       - "#approved-reviews-by>=1"
     actions:
       queue:
-        method: merge
+        method: squash
```

Comment:
I think that we need to put 
strict: true

The default value is false:
https://docs.mergify.com/actions/merge/#options
"false disables Strict Merge and merge pull requests as soon as possible, without bringing the pull request up-to-date with its base branch."